app/helpers/feature_helper.rb

```diff
@@ -0,0 +1,26 @@
+module FeatureHelper
+  def get_product_features(features_hash)
+    feature_klass = collection_class(:features)
+
+    new_features = []
+    features_hash.each do |feature|
```

Comment:
what kind of object is `features_hash`? i expected it to yield `[key, value]` pairs to `each`. what do you think about renaming this to clarify a bit?